src/utils/cleanupFilesForSurf.ts

```diff
@@ -0,0 +1,42 @@
+import fs from "node:fs";
+import { Selections } from "../types.js";
+import { TemplateFramework } from "./constants.js";
+import { move, remove } from "./helpers.js";
+
+export const cleanupFilesForSurf = (selection: Selections) => {
+  let frontend_dir: string;
+  if (selection.framework === TemplateFramework.VITE) {
+    frontend_dir = "frontend";
+  } else if (selection.framework === TemplateFramework.NEXTJS) {
+    frontend_dir = "src";
+  } else {
+    throw new Error("Unsupported framework");
+  }
+
+  if (selection.useSurf) {
```

Comment:
any reason not to add this check before entering the function? https://github.com/aptos-labs/create-aptos-dapp/pull/246/files#diff-0594d46782df78830c5efe0fa4abb36cbf22c918dc182ccc8f3cf75fdee908e7R87

Reply:
we need to delete file if not using surf